modules/charon-utils/src/main/java/org/wso2/charon3/utils/usermanager/InMemoryUserManager.java

```diff
@@ -80,12 +75,16 @@ public void deleteUser(String id)
     }
 
     @Override
-    public List<Object> listUsersWithGET(Node rootNode, int startIndex, int count, String sortBy,
-                                         String sortOrder, String domainName, Map<String, Boolean> requiredAttributes)
-            throws CharonException, NotImplementedException, BadRequestException {
+    public List<Object> listResources(Node rootNode,
```

Comment:
It is not recommended to change public method signatures as they can be used from some other places as well. We need to preserve the backward compatibility.